### PR TITLE
Update BBEs

### PR DESCRIPTION
**Pull request by Bhashinee**
## Purpose
Update BBEs

### CODE REVIEW

File: examples/https-listener/https_listener.bal [hunk 1]
```diff
@@ -4,7 +4,8 @@ import ballerina/log;
 
 // An HTTP endpoint can be configured to communicate through HTTPS as well.
 // To secure an endpoint using HTTPS, the endpoint needs to be configured with
-// a keystore, a certificate, and a private key for the endpoint.
+// a keyStore, and its `password` for the endpoint.
```
Comment (praneesha) [inline examples/https-listener/https_listener.bal:7]:
```suggestion
// a keystore and its `password` for the endpoint.
```

File: examples/https-listener/https_listener.bal [hunk 2]
```diff
@@ -4,7 +4,8 @@ import ballerina/log;
...
+// [secureSocket](https://ballerina.io/learn/api-docs/ballerina/http/records/ListenerSecureSocket.html) record provides the SSL related listener configurations.
```
Comment (praneesha) [inline examples/https-listener/https_listener.bal:8]:
```suggestion
// The [secureSocket](https://ballerina.io/learn/api-docs/ballerina/http/records/ListenerSecureSocket.html) record provides the SSL-related listener configurations.
```

File: examples/basic-https-listener-client/basic_https_listener_client.metatags [hunk 3]
```diff
@@ -0,0 +1,2 @@
+description: BBE on how the Ballerina https client can be configured to connect to an https listener through 1-way SSL connection(i.e.,the server is verified by the client).
```
Comment (Samallama) [inline examples/basic-https-listener-client/basic_https_listener_client.metatags:1]: Edit: BBE on how the Ballerina HTTPS client can be configured to connect to an HTTPS listener through 1-way SSL connection (i.e. the server is verified by the client)

(Added correct spacing, removed some punctuation to keep within the character limit)

File: examples/basic-https-listener-client/basic_https_listener_client.metatags [hunk 4]
```diff
@@ -0,0 +1,2 @@
...
+keywords: ballerina, ballerina by example, bbe, ssl, tls, https
```
Comment (Samallama) [inline examples/basic-https-listener-client/basic_https_listener_client.metatags:2]: Shall we add these: ballerina, ballerina by example, bbe, ssl, tls, https client, https listener

File: examples/https-listener/https_listener.metatags [hunk 5]
```diff
@@ -0,0 +1,2 @@
+description: BBE on how to use the Ballerina HTTPS Listener to connect or interact with an HTTPS client.
+keywords: ballerina, ballerina by example, bbe, ssl, https
```
Comment (Samallama) [inline examples/https-listener/https_listener.metatags:2]: Shall we add "https listener" as a keyword?

File: examples/request-with-multiparts/request_with_multiparts.metatags [hunk 6]
```diff
@@ -0,0 +1,2 @@
+description: BBE on how Ballerina supports encoding and decoding multipart content in http requests along with nested parts.
```
Comment (Samallama) [inline examples/request-with-multiparts/request_with_multiparts.metatags:1]: Edit: BBE on how Ballerina supports encoding and decoding multipart content in HTTP requests along with nested parts.

**Pull request merged**